lib/modules/platform/gitlab/http.ts

```diff
@@ -1,13 +1,22 @@
+import is from '@sindresorhus/is';
 import { logger } from '../../../logger';
 import { GitlabHttp } from '../../../util/http/gitlab';
 import type { GitLabUser, GitlabUserStatus } from './types';
 
 export const gitlabApi = new GitlabHttp();
 
 export async function getUserID(username: string): Promise<number> {
-  return (
+  const userInfo = (
     await gitlabApi.getJson<{ id: number }[]>(`users?username=${username}`)
-  ).body[0].id;
+  ).body;
+
+  if (!is.nonEmptyArray(userInfo)) {
```

Comment:
I would invert the condition for cleaner code

Reply:
```suggestion
  if (is.emptyArray(userInfo)) {
```